src/Model/Donor.php

```diff
@@ -71,6 +151,91 @@ public function getMandateSource(): string
         return $this->mandateSource;
     }
 
+    public function getPayerNumber(): string
+    {
+        return $this->payerNumber;
+    }
+
+    public function setPayerNumber(string $payerNumber)
+    {
+        $this->payerNumber = $payerNumber;
+    }
+
+    public function getAccount(): \byrokrat\banking\AccountNumber
+    {
+        return $this->account;
+    }
+
+    public function getId(): \byrokrat\id\Id
```

Comment:
`getId()` and `getUid()` is a little confusing. What do you think about `getDonorId()` and `getMandateKey()`?